src/pages/recruitment/recruitment.jsx

```diff
@@ -0,0 +1,135 @@
+import React, { Fragment, useState } from 'react';
+import styled from 'styled-components';
+import config from './formConfig';
+
+const FormWrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  width: 50%;
+`;
+
+const InputWrapper = styled.div`
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  margin: 20px 0;
+`;
+
+const Input = styled.input`
+  margin-left: 10px;
+  outline: none;
+
+  border-color: ${(props) => props.inValid && 'red'}
+`;
+
+const ErrorMessage = styled.div`
+  color: red;
+  display: flex;
+  justify-content: flex-end;
+`;
+
+const SubmitButton = styled.button`
+  width: 100%;
+
+  &:hover {
+    cursor: pointer;
+  }
+`;
+
+export default function RecruitmentPage() {
+  const [form, updateForm] = useState(
+    Object.keys(config).reduce(
+      (formObject, key) => ({
+        ...formObject,
+        [key]: '',
+      }),
+      {},
+    ),
+  );
```

Comment:
这里有点太长了不够 readable，要不要试试 
```suggestion
  const initialForm = Object.keys(config).reduce((formObject, key) => ({
    ...formObject,
    [key]: '',
  }), {});

  const [form, updateForm] = useState(initialForm);
```